app/javascript/mastodon/features/hashtag_timeline/components/hashtag_header.jsx

```diff
@@ -69,7 +84,7 @@ export const HashtagHeader = injectIntl(({ tag, intl, disabled, onClick }) => {
       </div>
     </div>
   );
-});
+}));
 
 HashtagHeader.propTypes = {
```

Comment:
These props might be a little "weird" since the HashtagHeader is with the intl injected, so intl wouldn't be a prop, same reason I didn't add `identity` here as a prop.